beta-src/src/utils/map/getUnits.ts

```diff
@@ -46,9 +46,7 @@ export default function getUnits(
               occupiedTerritory?.ownerCountryID !== unit.countryID &&
               territoryHasMultipleUnits.length > 1 &&
               phase === "Retreats") ||
-            (phase === "Retreats" &&
-              !territoryStatus?.occupiedFromTerrID &&
-              territoryHasMultipleUnits.length === 1) ||
+            (phase === "Retreats" && territoryHasMultipleUnits.length === 1) ||
```

Comment:
you already have this check on L47. Can we just do.

`territoryHasMultipleUnits.length > = 1`

Reply:
That first check is separate, its checking for territories that are occupied (have been moved into by another unit from the last phase) and has more than one unit in it. This second check ensures that any unit that moves into a territory that has no units before get drawn (the territory only has the mover unit in it so no disbanding or retreating action).